Vue Component.sublime-syntax

```diff
@@ -102,8 +102,8 @@ contexts:
               scope: punctuation.definition.tag.end.html
               set:
                 - include: style-close-tag
-                - embed_scope: source.sass.embedded.html
-                  match: ''
+                - match: ''
```

Comment:
All those contexts should follow the scheme of https://github.com/sublimehq/Packages/blob/c70117be6402c54ccf860ab8edfcba8aade46c9e/ASP/HTML%20(ASP).sublime-syntax#L45-L67 in order to not regress comment toggling in all those embedded syntaxes.

Reply:
Is that approach that you are suggesting here compatible with pre-4114 builds?

Reply:
Good point.

It should be enough to copy the variable declarations from current HTML.sublime-syntax to make them available in older builds and apply the scheme of those marked lines to JS and CSS.

Those marked contexts are the only ones which are involved in "fixing the toggle comment bug".

In case one of your embedded syntaxes supports `-->` as operator, you might need a custom strategy or just omit support for wrapping the script into html comments.

Reply:
Fingers crossed. I hope we don't need to touch it again.